lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -18,6 +18,7 @@ class CourseOverviewField(serializers.RelatedField):
     def to_representation(self, course_overview):
         course_id = unicode(course_overview.id)
         request = self.context.get('request', None)
+        discussion_url = None
```

Comment:
Please follow the coding pattern within this function and add this line to the `else` clause below.